src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -121,6 +121,24 @@ public boolean clickEditInstructorLink(int instrNum) {
         return isEditable;
     }
     
+    public void clickCancelEditInstructorLink(int instrNum) {
+        getCancelEditInstructorLink(instrNum).click();
+    }
+    
+    public boolean isInstructorFormNotEditable(int instrNum) {
+        WebElement saveButton = getSaveInstructorButton(instrNum);
+        waitForElementToDisappear(By.id("btnSaveInstructor" + instrNum));
```

Comment:
Yep that could be possible.

On a related note, does having this line (the `waitForElementToDisappear`) mean that it times out (and throws a `TimeoutException`) if the save button is present, and this method is unlikely to properly return a `false` value?